ingestion/src/metadata/ingestion/sink/elasticsearch.py

```diff
@@ -223,70 +209,89 @@ def __init__(
             self._check_or_create_index(
-                self.config.dashboard_index_name, DASHBOARD_ELASTICSEARCH_INDEX_MAPPING
+                ES_INDEX_MAP[Dashboard.__name__], DASHBOARD_ELASTICSEARCH_INDEX_MAPPING
             )
         if self.config.index_pipelines:
             self._check_or_create_index(
-                self.config.pipeline_index_name, PIPELINE_ELASTICSEARCH_INDEX_MAPPING
+                ES_INDEX_MAP[Pipeline.__name__], PIPELINE_ELASTICSEARCH_INDEX_MAPPING
             )
 
         if self.config.index_users:
             self._check_or_create_index(
-                self.config.user_index_name, USER_ELASTICSEARCH_INDEX_MAPPING
+                ES_INDEX_MAP[User.__name__], USER_ELASTICSEARCH_INDEX_MAPPING
             )
 
         if self.config.index_teams:
             self._check_or_create_index(
-                self.config.team_index_name, TEAM_ELASTICSEARCH_INDEX_MAPPING
+                ES_INDEX_MAP[Team.__name__], TEAM_ELASTICSEARCH_INDEX_MAPPING
             )
 
         if self.config.index_glossary_terms:
             self._check_or_create_index(
-                self.config.glossary_term_index_name,
+                ES_INDEX_MAP[GlossaryTerm.__name__],
                 GLOSSARY_TERM_ELASTICSEARCH_INDEX_MAPPING,
             )
 
         if self.config.index_mlmodels:
             self._check_or_create_index(
-                self.config.mlmodel_index_name,
+                ES_INDEX_MAP[MlModel.__name__],
                 MLMODEL_ELASTICSEARCH_INDEX_MAPPING,
             )
 
         if self.config.index_tags:
             self._check_or_create_index(
-                self.config.tag_index_name,
+                ES_INDEX_MAP[Tag.__name__],
                 TAG_ELASTICSEARCH_INDEX_MAPPING,
             )
 
         if self.config.index_entity_report_data:
             self._check_or_create_index(
-                self.config.entity_report_data_index_name,
+                ES_INDEX_MAP[ReportData.__name__],
                 ENTITY_REPORT_DATA_INDEX_MAPPING,
             )
 
         if self.config.index_web_analytic_user_activity_report_data:
             self._check_or_create_index(
-                self.config.web_analytic_user_activity_report_data_index_name,
+                ES_INDEX_MAP["web_analytic_user_activity_report"],
                 WEB_ANALYTIC_USER_ACTIVITY_REPORT_DATA_INDEX_MAPPING,
             )
 
         if self.config.index_web_analytic_entity_view_report_data:
             self._check_or_create_index(
-                self.config.web_analytic_entity_view_report_data_name,
+                ES_INDEX_MAP["web_analytic_entity_view_report"],
                 WEB_ANALYTIC_ENTITY_VIEW_REPORT_DATA_INDEX_MAPPING,
             )
 
+        if self.config.index_containers:
+            self._check_or_create_index(
+                ES_INDEX_MAP[Container.__name__],
+                CONTAINER_ELASTICSEARCH_INDEX_MAPPING,
+            )
+
+        if self.config.index_queries:
```

Comment:
all these if-else will disappear in https://github.com/open-metadata/OpenMetadata/issues/9185